src/coupling/balance.hh

```diff
@@ -471,7 +476,10 @@ private:
     /// Vectors for calculation of mass (n_bulk_regions).
     Vec *region_mass_vec_;
 
-    /// Number of boundary region for each local boundary edge.
+    /// Maps unique identifier of (local bulk element idx, side idx) to local boundary edge.
```

Comment:
Improve documentation so that it is clear, that the LongIdx key is the result of get_boundary_edge_uid(s), where s is a side of a locally owned element.